multichain-testing/scripts/deploy-cli.ts

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env tsx
+import '@endo/init/debug.js';
+
+import { execa } from 'execa';
+import fse from 'fs-extra';
+import childProcess from 'node:child_process';
+
+import { makeAgdTools } from '../tools/agd-tools.js';
+import { makeDeployBuilder } from '../tools/deploy.js';
+
+async function main() {
```

Comment:
I expect we'll DRY later

Reply:
Nothing similar but I could see it being a regular CLI command. Though I didn't have any specific ideas in mind other than to ignore my DRY/factoring sniffer.